src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,13 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[3];
+        Lottery lottery = new Lottery();
+
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = lottery.getRandomBall();
+            System.out.println(balls[i].getColor() + " ball with number "
+                    + balls[i].getNumber());
+        }
```

Comment:
no need to use array
create `toString()` method in `Ball` class
```suggestion
        Lottery lottery = new Lottery();

        for (int i = 0; i <3; i++) {
            System.out.println(lottery.getRandomBall());
        }
```